packages/server/src/tests/jestSetup.ts

```diff
@@ -1,9 +1,6 @@
-import { mocks } from "@budibase/backend-core/tests"
-
-// mock all dates to 2020-01-01T00:00:00.000Z
-// use tk.reset() to use real dates in individual tests
-const tk = require("timekeeper")
-tk.freeze(mocks.date.MOCK_DATE)
```

Comment:
Can we re-add the date freezing here? I find frozen dates by default are useful, especially around making tests and data more predictable. It's also consistent with how it's done in the other packages. 

We can still turn them off when required by using `tk.reset()` in a particular test